AnnotatoBackend/Sources/App/Routes/WebSocketRouter.swift

```diff
@@ -0,0 +1,7 @@
+import Vapor
+
+func webSocketRouter(ws: RoutesBuilder) {
+    ws.group(":id") { user in
```

Comment:
```suggestion
    ws.group(":userId") { user in
```

As it's currently not clear what `id` is referring to since it's not in any entity's route group